docs/schema.json

```diff
@@ -17,10 +17,6 @@
       "title": "Network settings",
       "description": "Basic settings of the Hyperledger Fabric network",
       "type": "object",
-      "default": {
-        "tls": false,
-        "fabricVersion": "1.4.4"
-      },
```

Comment:
hm... why default was removed?

Reply:
because it is already specified for fields in lines 50/41 and 56/47